content/docs/guides/troubleshooting.md

```diff
@@ -17,6 +17,16 @@ Linux:
 grep -ci avx /proc/cpuinfo
 ```
 
+### AVX2
+Some components requirement AVX2 for better performance, e.g. TF Serving.
+To ensure the nodes support AVX2, we added minCpuPlatform arg in our deployment
+[config](https://github.com/kubeflow/kubeflow/blob/master/scripts/gke/deployment_manager_configs/cluster.jinja#L105).
+
+This will fail in regions (e.g. us-central1-a) that do not have Intel Haswell.
+In that case, please choose another region, or change the config to other 
+[platform](https://en.wikipedia.org/wiki/List_of_Intel_CPU_microarchitectures)
+newer than Haswell.
```

Comment:
Are you saying that we won't  run in us-central1-a with Haswell; but if we pick a newer platform then we can run in us-central1a?

Reply:
I think so. https://cloud.google.com/compute/docs/regions-zones/#available
us-central1-a has skylake which is newer than haswell